tarodb/sqlite/migrations/000002_assets.up.sql

```diff
@@ -218,7 +218,7 @@ CREATE TABLE IF NOT EXISTS asset_seedlings (
 
     -- TODO(roasbeef): data redundant w/ genesis_assets?
     -- move into asset details table?
-    asset_name TEXT NOT NULL,
+    asset_name TEXT UNIQUE NOT NULL,
```

Comment:
Does this address https://github.com/lightninglabs/taro/issues/119 as is? I would think that we'd need to pass thru a new error somewhere? 

I think would be useful to add a basic itests here (creating asset w/ the same name in a batch is rejected), as a regression to ensure that the crash is actually fixed.

Reply:
That change alone doesn't address https://github.com/lightninglabs/taro/issues/119 fully. I've forced push a new commit which includes go code for verifying candidate seedlings. I think it's better not to solely depend on a database constraint.

I agree that an itest would be a good idea. I'll add that next.